RecoJets/JetProducers/interface/ECFAdder.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ECFAdder_h
+#define ECFAdder_h
+
+#include <memory>
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "DataFormats/JetReco/interface/Jet.h"
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "fastjet/contrib/EnergyCorrelator.hh"
+
+
+class ECFAdder : public edm::EDProducer { 
```

Comment:
this should be made multithreading-friendly
choice will depending on thread-safety of the EnergyCorrelator.
@Dr15Jones is it on our bad or good list?